### ISSUE TITLE
The point cloud scale.

### ISSUE THREAD

**Issue by wutong16** [xiaogangw-zz/cascaded-point-completion#11]
Hi! Thank you for your great work!

I find that in the **train set**, the scale of the incomplete points clouds could sometimes be larger than its corresponding ground truth (as shown in the figure below, where red refers to the partial point could and blue refers to its ground truth, both without augmentation). This does not happen to the **test set**. I'm not sure if there is something wrong with my usage of the dataset or if this phenomenon does exist. 
Could you provide some ideas on that? Many thanks!
![image](https://user-images.githubusercontent.com/28827385/109589601-8ccae880-7b45-11eb-864a-801e8f465059.png)

**Comment by yjcaimeow** [comment]
+1 the same question

**Comment by stasinak** [comment]
Hi @wutong16, I came to the same conclusion after visualizing some of the training objects. Therefore, I guess this is not a problem with your usage of the data. In any case, are you aware of the pre-processing step that the authors are using in terms (mostly) of the normalization of each point cloud? There is not any code for the preprocessing and I did not find any comment on how they normalize the data.

**Comment by wutong16** [comment]
Hi @stasinak, I'm not aware of the normalization method either...

**Comment by xiaogangw** [comment]
@wutong16 @stasinak @yjcaimeow Thanks for your interest in our work. I also noticed there are some scale differences in the partial and complete points in our training data, but we **_do not_** do any special normalization, which we follow PCN. Network itself is robust enough with this. Moreover, our training samples are directly selected from PCN's dataset, where we just selected one partial view instead of eight views in PCN for each object instance. We just want to test models' generalization abilities with our fewer training data. Hope this clarifies.
